cico_build_deploy_npm.sh

```diff
@@ -20,7 +20,7 @@ yum clean all
 service docker start
 
 docker build -t fabric8-planner-builder -f Dockerfile.builder .
-mkdir -p dist && docker run --detach=true --name=fabric8-planner-builder -e "API_URL=http://api.prod-preview.openshift.io/api/" -e JENKINS_URL -e GIT_BRANCH -e "CI=true" -e GH_TOKEN -e NPM_TOKEN -t -v $(pwd)/dist:/dist:Z fabric8-planner-builder
+mkdir -p dist && docker run --detach=true --name=fabric8-planner-builder -e "API_URL=https://api.prod-preview.openshift.io/api/" -e JENKINS_URL -e GIT_BRANCH -e "CI=true" -e GH_TOKEN -e NPM_TOKEN -t -v $(pwd)/dist:/dist:Z fabric8-planner-builder
```

Comment:
But maybe another commit and PR, since this PR is all about changing the http to https, we can keep this, can't we?